creator-node/src/routes/nodeSync.js

```diff
@@ -305,21 +305,22 @@ async function _nodesync (req, walletPublicKeys, creatorNodeEndpoint) {
       } else throw new Error(`Malformed response from ${creatorNodeEndpoint}.`)
```

Comment:
does line 298 above also need to change? i just noticed it

Reply:
Actually, no -- 'error' is a top level directive:
https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/apiHelpers.js#L163

vs. the nested data field 
https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/apiHelpers.js#L139

Reply:
ah perf, i prob should have just checked this myself my b